experiments/ssl4eo/download_l8_l2.sh

```diff
@@ -22,6 +22,7 @@ YEAR=2022
 BANDS=(SR_B1 SR_B2 SR_B3 SR_B4 SR_B5 SR_B6 SR_B7)
 ORIGINAL_RESOLUTIONS=30
 NEW_RESOLUTIONS=30
+DEFAULT_VALUE=0
```

Comment:
the python script says `default_value` is supposed to be `float` but this is integer, or do you not use floats in bash scripts?